Program.cs

```diff
@@ -100,10 +116,18 @@ class UrwObject {
 			this.valuePerLbs = data.ReadSingle();
 
 			data.ReadBytes(9); // Unknown
-		
+
 			this.quality = (byte)(data.ReadByte() & 0x0F);
 
-			data.ReadBytes(6); // Unknown
+			data.ReadBytes(1); // Unknown
+
+			if (type == UrwObjectType.Weapon) {
+				int combatBonuses = data.ReadByte();
+				this.attackBonus = (byte)(combatBonuses & 0x0F);
+				this.defenceBonus = (byte)(combatBonuses >> 4);
+			}
```

Comment:
You are right, I will add it. That "if" was a last minute addition (superfluous really) which I hadn't tested.